src/huggingface_hub/hf_api.py

```diff
@@ -7746,6 +7763,8 @@ def update_inference_endpoint(
         min_replica: Optional[int] = None,
         max_replica: Optional[int] = None,
         scale_to_zero_timeout: Optional[int] = None,
+        scaling_metric: Optional[str | EndpointsScalingMetric] = None,
```

Comment:
same comment as the one in `create_inference_endpoint`
```suggestion
        scaling_metric: Optional[EndpointsScalingMetric] = None,
```